tests/functional/xtriggers/04-suite_state.t

```diff
@@ -29,7 +29,7 @@ init_workflow "$TEST_NAME_BASE" << __FLOW_CONFIG__
         [[[R1]]]
             graph = @upstream => foo
     [[xtriggers]]
-        upstream = suite_state(suite=thorin/oin/gloin, task=mithril, point=1)
+        upstream = suite_state(suite=thorin/oin/gloin, task=mithril, point=1, status="succeeded")
```

Comment:
I don't think this change is needed?

Reply:
Good spot - it briefly was, early in branch development.